### DISCUSSION TITLE
OpenEDR Integration

### DISCUSSION THREAD

**Discussion by DrStupendous** [Security-Onion-Solutions/securityonion#10643] [Unsupported Versions] [answered]
Hello Security Onion pro's
I was wondering if there is a way to integrate openEDR into security onion for Endpoint response capabilities?

Or if there is any opensource EDR that can be integrated into Security Onion?

**Comment by InfosecGoon** [comment, marked as answer]
For endpoint response, would Velociraptor be sufficient? This might interest you:

https://github.com/weslambert/securityonion-velociraptor

**Comment by DrStupendous** [comment]
Thank you,

I was looking into velociraptor aswell.. I will throw that in a test deployment
